classes/Tab.php

```diff
@@ -316,6 +316,14 @@ public static function getTabs($idLang, $idParent = null)
 
             if (is_array($result)) {
                 foreach ($result as $row) {
+                    if (!empty($row['module'])) {
+                        $module = Module::getInstanceByName($row['module']);
+                        //not to show the admin tabs of the deactivated modules
+                        if (Validate::isLoadedObject($module) && !$module->isEnabledForShopContext()) {
+                            continue;
```

Comment:
What do you think about modifying the SQL query directly so we don't need to make an SQL request for each module that has a tab?